frontends/api/src/test-utils/mockAxios.ts

```diff
@@ -68,6 +68,15 @@ const mockAxiosInstance = {
  */
 const expectAnything: unknown = when(() => true)
 
+const standardizeUrl = <T>(url: T) => {
```

Comment:
I couldn't quite grasp the intention of this, given the tests and `queryify` don't seem to be guaranteeing sort order in the params.

Reply:
The idea was that jest would match against a sorted URL. Sorting in querify is a better idea, though.